File: CyclepathAPI/Models/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace CyclepathAPI.Models
{
    public class Event
    {
        [Key]
        [Column(Order = 1)]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public int OwnerId { get; set; }
        public string Name { get; set; }
        public string EventDate { get; set; }
        public int RouteId { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
    }
}

File: CyclopathIU/src/models/event.ts
export class Event {
    id: number;
    ownerId: number;
    name: string;
    latitude: number;
    longitude: number; 
    routeId: number; 
    eventDate: string;
}

export interface IEvent {
    id: number;
    ownerId: number;
    name: string;
    latitude: number;
    longitude: number; 
    routeId: number; 
    eventDate: string;
}

export interface IEnlistment {
    id: number;
    idAccount: number;
    idEvent: number;
}